articles/connections/apple-siwa/add-siwa-to-native-app.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Add Sign In with Apple to Native iOS Apps
+description: Learn how to add native login functionality to your native app with Apple. 
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Native iOS Apps
+
+You can add functionality to your native iOS application that allows your users to use Sign In with Apple to authenticate. In addition, you can also try the Auth0 [iOS Swift - Sign In with Apple Quickstart](/quickstart/native/ios-swift-siwa).
+
+## How it works
+
+For a native app, the Sign in with Apple login flow works as follows:
+
+![Sign In with Apple Authentication Flow](/media/articles/connections/social/apple/apple-siwa-authn-flow.png)
+
+* User authenticates via Apple's SDK on their iPhone or iPad. They receive an authorization code. The user does not have to leave the app and use a browser to log in.
+* The application calls Auth0's `/oauth/token` endpoint with the following parameters:
```

Comment:
+1 for including this - had a note to make sure it was called out for those who aren't using the Swift SDK.  Nice!